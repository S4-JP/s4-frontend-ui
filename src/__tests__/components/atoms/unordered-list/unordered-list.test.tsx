import React from 'react'
import { render } from '@testing-library/react'
import { UnorderedList } from '../../../../index'

const Dummy = () => <span>dummy</span>
describe('UnorderedListのレンダリング確認', () => {
  test('propsに設定した値でレンダリングされる。', () => {
    const { getByTestId } = render(
      <UnorderedList>
        <Dummy />
      </UnorderedList>
    )

    expect(getByTestId('atoms-ul-text')).toMatchSnapshot()
  })
})
