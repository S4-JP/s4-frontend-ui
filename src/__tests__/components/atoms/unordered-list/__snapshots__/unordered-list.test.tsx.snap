// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnorderedListのレンダリング確認 propsに設定した値でレンダリングされる。 1`] = `
<li
  class="sc-jQHtVU fNzYKJ"
  data-testid="atoms-ul-text"
>
  <span>
    dummy
  </span>
</li>
`;
